runtime/src/lib.rs

```diff
@@ -236,31 +255,172 @@ impl pallet_sudo::Config for Runtime {
+    type CurrencyId = Asset;
+    type Event = Event;
+    type ExistentialDeposits = ExistentialDeposits;
+    type OnDust = orml_tokens::BurnDust<Runtime>;
+    type WeightInfo = ();
+}
+
+parameter_types! {
+    pub const ProxyDepositBase: Balance = 1;
+    pub const ProxyDepositFactor: Balance = 1;
+    pub const MaxProxies: u16 = 4;
+    pub const MaxPending: u32 = 2;
+    pub const AnnouncementDepositBase: Balance = 1;
+    pub const AnnouncementDepositFactor: Balance = 1;
+    pub const GetUsdvId: Asset = Asset::Usdv;
+}
+
+impl pallet_proxy::Config for Runtime {
     type Event = Event;
-    type OnValidationData = ();
-    type SelfParaId = parachain_info::Module<Runtime>;
-    type DownwardMessageHandlers = ();
-    type HrmpMessageHandlers = ();
+    type Call = Call;
+    type Currency = orml_tokens::CurrencyAdapter<Runtime, GetUsdvId>;
+    type ProxyType = ProxyType;
+    type ProxyDepositBase = ProxyDepositBase;
+    type ProxyDepositFactor = ProxyDepositFactor;
+    type MaxProxies = MaxProxies;
+    type WeightInfo = ();
+    type CallHasher = BlakeTwo256;
+    type MaxPending = MaxPending;
+    type AnnouncementDepositBase = AnnouncementDepositBase;
+    type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
-impl parachain_info::Config for Runtime {}
-
-// Create the runtime by composing the FRAME pallets that were previously configured.
-construct_runtime!(
-    pub enum Runtime where
-        Block = Block,
-        NodeBlock = opaque::Block,
-        UncheckedExtrinsic = UncheckedExtrinsic,
-    {
-        System: frame_system::{Module, Call, Storage, Config, Event<T>},
-        Timestamp: pallet_timestamp::{Module, Call, Storage, Inherent},
-        Sudo: pallet_sudo::{Module, Call, Storage, Config<T>, Event<T>},
-        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Module, Call, Storage},
-        ParachainSystem: cumulus_pallet_parachain_system::{Module, Call, Storage, Inherent, Event},
-        ParachainInfo: parachain_info::{Module, Storage, Config},
+#[cfg(feature = "standalone")]
+pub use standalone_impl::*;
+
+#[cfg(feature = "standalone")]
+mod standalone_impl {
+    use super::*;
+
+    impl_opaque_keys! {
+        pub struct SessionKeys {
+            pub aura: Aura,
+            pub grandpa: Grandpa,
+        }
     }
-);
+
+    impl pallet_aura::Config for Runtime {
+        type AuthorityId = AuraId;
+    }
+
+    impl pallet_grandpa::Config for Runtime {
+        type Call = Call;
+        type Event = Event;
+        type KeyOwnerProofSystem = ();
+        type KeyOwnerProof =
+            <Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(KeyTypeId, GrandpaId)>>::Proof;
+        type KeyOwnerIdentification = <Self::KeyOwnerProofSystem as KeyOwnerProofSystem<(
+            KeyTypeId,
+            GrandpaId,
+        )>>::IdentificationTuple;
+        type HandleEquivocation = ();
+        type WeightInfo = ();
+    }
+
+    impl vln_foreign_asset::Config for Runtime {
```

Comment:
Are our pallets required to be inside the feature gated block?

Reply:
Not really, I thought will do that in the next PR along with integrating xcm, but no problem, I will add it here

Reply:
Fixed in https://github.com/valibre-org/vln-node/pull/69/commits/6d72be08eb4d6d439473c3bc9349d8e364fa1ecc